Biz/Model/RealmOwner.pm

```diff
@@ -333,20 +337,25 @@ sub update_password {
     });
 }
 
-sub validate_login {
-    my($self, $login) = @_;
-    my($err) = !$self->unauth_load_by_email_id_or_name($login) ? ''
-        : $self->is_offline_user ? 'is_offline_user'
+sub validate {
```

Comment:
`validate` is too short. `validate_login_for_self'? Or validate_login becomes load_and_validate_login and validate becomes validate_login, but this cascades so better to keep validate_login as API unchanged.